src/assets/scss/abstracts/_mobile.scss

```diff
@@ -0,0 +1,27 @@
+//Small Devices
+@mixin mediaSm {
```

Comment:
Good job for this file. I recommend it to all of teammates to use it since they merge this PR.